{"cause":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{"localhost:27017":{"$clusterTime":null,"address":"localhost:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":602626792,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"}},"setName":null,"stale":false,"type":"Unknown"},"errorLabelSet":{},"level":"error","message":"MongoDB connection error: connect ECONNREFUSED ::1:27017, connect ECONNREFUSED 127.0.0.1:27017","reason":{"commonWireVersion":0,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{"localhost:27017":{"$clusterTime":null,"address":"localhost:27017","arbiters":[],"electionId":null,"error":{"beforeHandshake":false,"errorLabelSet":{}},"hosts":[],"iscryptd":false,"lastUpdateTime":602626792,"lastWriteDate":0,"logicalSessionTimeoutMinutes":null,"maxBsonObjectSize":null,"maxMessageSizeBytes":null,"maxWireVersion":0,"maxWriteBatchSize":null,"me":null,"minRoundTripTime":0,"minWireVersion":0,"passives":[],"primary":null,"roundTripTime":-1,"setName":null,"setVersion":null,"tags":{},"topologyVersion":null,"type":"Unknown"}},"setName":null,"stale":false,"type":"Unknown"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017, connect ECONNREFUSED 127.0.0.1:27017\n    at _handleConnectionErrors (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongoose\\lib\\connection.js:1165:11)\n    at NativeConnection.openUri (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongoose\\lib\\connection.js:1096:11)","timestamp":"2025-09-10T07:24:20.158Z"}
{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"errorLabelSet":{},"errorResponse":{"code":8000,"codeName":"AtlasError","errmsg":"bad auth : authentication failed","ok":0},"level":"error","message":"MongoDB connection error: bad auth : authentication failed","ok":0,"stack":"MongoServerError: bad auth : authentication failed\n    at Connection.sendCommand (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:305:27)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:333:26)\n    at async continueScramConversation (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:131:15)\n    at async executeScram (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:80:5)\n    at async ScramSHA1.auth (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:39:16)\n    at async performInitialHandshake (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:104:13)\n    at async connect (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:24:9)","timestamp":"2025-09-10T07:31:21.113Z"}
{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"errorLabelSet":{},"errorResponse":{"code":8000,"codeName":"AtlasError","errmsg":"bad auth : authentication failed","ok":0},"level":"error","message":"MongoDB connection error: bad auth : authentication failed","ok":0,"stack":"MongoServerError: bad auth : authentication failed\n    at Connection.sendCommand (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:305:27)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:333:26)\n    at async continueScramConversation (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:131:15)\n    at async executeScram (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:80:5)\n    at async ScramSHA1.auth (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:39:16)\n    at async performInitialHandshake (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:104:13)\n    at async connect (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:24:9)","timestamp":"2025-09-10T07:32:36.262Z"}
{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"errorLabelSet":{},"errorResponse":{"code":8000,"codeName":"AtlasError","errmsg":"bad auth : authentication failed","ok":0},"level":"error","message":"MongoDB connection error: bad auth : authentication failed","ok":0,"stack":"MongoServerError: bad auth : authentication failed\n    at Connection.sendCommand (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:305:27)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:333:26)\n    at async continueScramConversation (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:131:15)\n    at async executeScram (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:80:5)\n    at async ScramSHA1.auth (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:39:16)\n    at async performInitialHandshake (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:104:13)\n    at async connect (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:24:9)","timestamp":"2025-09-10T07:32:44.627Z"}
{"code":8000,"codeName":"AtlasError","connectionGeneration":0,"errorLabelSet":{},"errorResponse":{"code":8000,"codeName":"AtlasError","errmsg":"bad auth : authentication failed","ok":0},"level":"error","message":"MongoDB connection error: bad auth : authentication failed","ok":0,"stack":"MongoServerError: bad auth : authentication failed\n    at Connection.sendCommand (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:305:27)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Connection.command (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connection.js:333:26)\n    at async continueScramConversation (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:131:15)\n    at async executeScram (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:80:5)\n    at async ScramSHA1.auth (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\auth\\scram.js:39:16)\n    at async performInitialHandshake (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:104:13)\n    at async connect (C:\\Users\\Virendra Wagh\\Documents\\Browser-atm-ai-tool\\node_modules\\mongodb\\lib\\cmap\\connect.js:24:9)","timestamp":"2025-09-10T07:32:55.714Z"}
